src/main/java/com/boardgamesworld/bgrental/services/UserService.java

```diff
@@ -11,52 +10,34 @@
 @Service
 public class UserService {
 
-    private UserRepository customerRepository;
-    private BoardGameRepository boardGameRepository;
+    private UserRepository userRepository;
+    private UserValidatorService userValidatorService;
 
     @Autowired
-    public UserService(UserRepository customerRepository, BoardGameRepository boardGameRepository) {
-        this.customerRepository = customerRepository;
-        this.boardGameRepository = boardGameRepository;
+    public UserService(UserRepository userRepository, UserValidatorService userValidatorService) {
+        this.userRepository = userRepository;
+        this.userValidatorService = userValidatorService;
     }
 
     public List<User> getAllUser() {
-        return customerRepository.getAllUser();
+        return userRepository.getAllUser();
     }
 
     public User getUser(long userId) {
-        return customerRepository.getUser(userId);
+        return userRepository.getUser(userId);
     }
 
     public void addUser(User user) {
-        customerRepository.addUser(user);
+        userValidatorService.isUserValid(user);
```

Comment:
isUserValid sugeruje ze zwrócisz booleana, a nigdzie nie przypisujesz wyniku do jakiejs zmiennej. 

MOże powinno to się nazywać `validate(user)`?
`userValidatorService` -> `userValidator` - nie ma co doklejac wszedzie na sile service